### PR TITLE
{vis}[intel/2019b] NiBabel v3.1.0 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, x86_64, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/59418966b0dfecdbc90ae469f62bdf9c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3102.skitty.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/7b5eba95f600a6fd31ed0bfc03af0b3f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2422.golett.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz (haswell), Python 2.7.5
See https://gist.github.com/b28bd1c0de09a47db1d0df5029550c96 for a full test report.

**Comment by zao** [comment]
Test report by @zao
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
freja - Linux Ubuntu 20.04, x86_64, Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz, Python 3.8.2
See https://gist.github.com/4feab7beeb84a720315f61aebf60fbb4 for a full test report.

File: easybuild/easyconfigs/n/Nipype/Nipype-1.4.2-intel-2019b-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,89 @@
+easyblock = 'PythonBundle'
+
+name = 'Nipype'
+version = '1.4.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://nipy.org/nipype'
```
Comment (Micket) [inline easybuild/easyconfigs/n/Nipype/Nipype-1.4.2-intel-2019b-Python-3.7.4.eb:7]:
```suggestion
homepage = 'https://nipype.readthedocs.io'
```

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in this PR)
node379.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/c40512be3e9684347ee200a2bff66066 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in this PR)
node127.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/7159e6edd2d28295cc543eb702b4decd for a full test report.

**Comment by lexming** [comment]
Going in, thanks @boegel !

**Pull request merged**